src/base/utils/net.h

```diff
@@ -28,13 +28,26 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
+#include <QList>
+
+class QHostAddress;
 class QString;
+class QStringList;
+template <typename T, typename U> struct QPair;
```

Comment:
This is a primitive type IMO, why forward declare it instead of including the header?